src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/Utils/PublicationAuthorizationHandlersTestUtil.cs

```diff
@@ -117,12 +120,83 @@ public static class PublicationAuthorizationHandlersTestUtil
             });
         }
         
+        public class PublicationHandlerTestScenario : HandlerTestScenario
```

Comment:
Might be worth looking at the way we check publication roles elsewhere, .e.g. in `CreateMethodologyForSpecificPublicationAuthorizationHandlerTests` where  we use a mock for the `PublicationRoleRepository`, and we can verify interactions with it and no need to set up any test data except setting up the behaviour of `IsUserPublicationOwner` which can only be true or false.

Just wondering if you spotted this but dismissed it before creating this approach?

Reply:
Discussed this on slack. I am going to leave it as it is for now.